### PR TITLE
fix: (helm) update shard values and set default

### PR DESCRIPTION
**Pull request by jgreat**
### Motivation

- set better router and shard values for test/main. 

I moved the shard range generator count into where the stores were configured for each environment and changed the name of the `store` section to `shardRangeGenerator` now that the settings cover more than just the stores.

### CODE REVIEW

File: .internal-ci/helm/fog-services/values.yaml [hunk 1]
```diff
@@ -41,18 +41,24 @@ ingress:
   enabled: true
 
 #### Fog View
-fogViewStores:
+fogViewShardRangeGenerator:
+  # The dev options are set in the github workflow deployment.
+  # This is just for reference.
+  # dev:
+  #   shardSize: 25_000
```
Comment (nick-mobilecoin) [inline .internal-ci/helm/fog-services/values.yaml:48]: Looking at .github/workflows/mobilecoin-workflow-dev-deploy.yaml
the values seem to be 10_000 for size and 5_000 for exceed by

File: .internal-ci/helm/fog-services/values.yaml [hunk 2]
```diff
@@ -152,18 +158,24 @@ fogView:
...
 #### Fog Ledger
-fogLedgerStores:
+fogLedgerShardRangeGenerator:
+  # The dev options are set in the github workflow deployment.
+  # This is just for reference.
+  # dev:
+  #   shardSize: 25_000
+  #   exceedBlockHeightBy: 10_000
+  #   shardOverlap: 0
+  #   count: 2
```
Comment (nick-mobilecoin) [inline .internal-ci/helm/fog-services/values.yaml:168]: I don't think I understand how the override is working in .github/workflows/mobilecoin-workflow-dev-deploy.yaml, but I get the feeling it's the same as the view l
the values seem to be 10_000 for size and 5_000 for exceed by

**Review by nick-mobilecoin — APPROVED** [review]

File: .internal-ci/helm/fog-services/values.yaml [hunk 3]
```diff
@@ -152,18 +158,24 @@ fogView:
...
   main:
-    shardSize: 400_000
-    exceedBlockHeightBy: 200_000
-    shardOverlap: 200_000
+    shardSize: 800_000
+    exceedBlockHeightBy: 10_000
```
[the remark this answers is not in the document]
Comment (jgreat) [reply .internal-ci/helm/fog-services/values.yaml:171]: Good catch. should be 400_000

File: .internal-ci/helm/fog-services/values.yaml (hunk 1 above)
Comment (jgreat) [reply .internal-ci/helm/fog-services/values.yaml:48]: I'll fix that reference

**Review by aweis89 — APPROVED** [review]
LGTM

**Pull request merged**